bpmn2stamp-converter/pom.xml

```diff
@@ -201,39 +201,36 @@
                 </configuration>
             </plugin>
             <!-- JOPA CLASS GENERATOR -->
-<!--            <plugin>-->
-<!--                &lt;!&ndash; Generate entity classes from OWL integrity constraints &ndash;&gt;-->
-<!--                <groupId>cz.cvut.kbss.jopa</groupId>-->
-<!--                <artifactId>jopa-maven-plugin</artifactId>-->
-<!--                <version>${cz.cvut.kbss.jopa.version}</version>-->
-<!--                <configuration>-->
-<!--                    &lt;!&ndash; STAMP &ndash;&gt;-->
+            <plugin>
+                <!-- Generate entity classes from OWL integrity constraints -->
+                <groupId>cz.cvut.kbss.jopa</groupId>
+                <artifactId>jopa-maven-plugin</artifactId>
+                <version>${cz.cvut.kbss.jopa.version}</version>
+                <configuration>
+                    <!-- Uncomment one of the following sections (only one at a time) to generate the corresponding model -->
+                    <!-- STAMP -->
 <!--                    <package>model.stamp</package>-->
 <!--                    <ontology-iri>http://onto.fel.cvut.cz/ontologies/stamp</ontology-iri>-->
-<!--                    &lt;!&ndash; BBO &ndash;&gt;-->
+                    <!-- BBO -->
 <!--                    <package>model.bbo</package>-->
 <!--                    <ontology-iri>http://BPMNbasedOntology</ontology-iri>-->
-<!--                    &lt;!&ndash; BBO-E &ndash;&gt;-->
+                    <!-- BBO-E -->
 <!--                    <package>model.bbo-e</package>-->
 <!--                    <ontology-iri>http://onto.fel.cvut.cz/ontologies/bbo</ontology-iri>-->
 
-<!--                    <output-directory>${project.basedir}/src/main/java</output-directory>-->
-<!--                    <mapping-file>${project.basedir}/src/main/resources/jopa/mapping/mapping.map</mapping-file>-->
-<!--&lt;!&ndash;                    <context-name>plan-execution</context-name>&ndash;&gt;-->
-<!--&lt;!&ndash;                    <whole-ontology-as-ics>true</whole-ontology-as-ics>&ndash;&gt;-->
-<!--&lt;!&ndash;                    <vocabulary-only>true</vocabulary-only>&ndash;&gt;-->
-<!--&lt;!&ndash;                    <ignore-failed-imports>true</ignore-failed-imports>&ndash;&gt;-->
-<!--                </configuration>-->
+                    <output-directory>${project.basedir}/src/main/java/cz/cvut/kbss/bpmn2stamp/converter</output-directory>
+                    <mapping-file>${project.basedir}/src/main/resources/jopa/mapping/mapping-owl2java-transform.map</mapping-file>
```

Comment:
please remove file mapping.map if you are not using it.

Reply:
unused file mapping.map was removed